documentation/userguide.md

```diff
@@ -306,6 +306,9 @@ uploader and in Teamscale. Patterns are glob patterns: `*` matches any number of
       include: [ "*YourAssembly*" ]
       exclude: [ "*DoNotProfileThisAssembly*" ]
 
+Please note that the profiler cannot create line-accurate coverage, but only method-accurate coverage.
+The generated coverage will therefore not map to specific line numbers, but to methods.
```

Comment:
I would probably move these sentences somewhere to the beginning of the file as this is essential information. This should probably be placed so that users can easily see it to prevent confusion. I would also add a reference to TGA and explain that method-accurate coverage is sufficient as TGA also operates on basis of methods instead of single lines. My suggestion would be to have this after the part "The profiler writes method coverage information of..." in the first section (lines 8ff):
```
The profiler writes method coverage information of a .NET application into a report file (a.k.a. *trace file*). It uses the builtin .NET profiling interface and therefore can be used with any .NET application.
Please note that the profiler does not create line-accurate coverage, but only method-accurate coverage.
The generated coverage will therefore not map to specific line numbers, but to methods.
This is sufficient for TGA in Teamscale which operates on the basis of methods.
```

Reply:
Great idea, thanks! Moved this and also added a sentence regarding the relation to TGA.